spyder/widgets/editor.py

```diff
@@ -1979,6 +2041,18 @@ def dropEvent(self, event):
                 editor.insert_text( source.text() )
         event.acceptProposedAction()
 
+    def keyReleaseEvent(self, event):
+        """Reimplement Qt method.
+
+        Handle "most recent used" tab behavior,
+        When ctrl is released and tab_switcher is visible, tab will be changed.
+        """
+        if (event.key() == Qt.Key_Control and self.tabs_switcher is not None

```

Comment:
I'll add an option for also considering Alt, I think the shortcuts validation could be addressed in other Issue in a more general and reusable way